.travis/before_script_test.sh

```diff
@@ -1,3 +1,10 @@
-#!/usr/bin/env bash
```

Comment:
Please revert the CHMOD

Reply:
You are right, I thought it was done by mistake.

We can keep this, I added a PR on dev-kit for the other bundle 👍 
https://github.com/sonata-project/dev-kit/pull/440